.changeset/dirty-wolves-sin.md

```diff
@@ -0,0 +1,5 @@
+---
+"@google/generative-ai": minor
+---
+
+Expand the model's countTokens method to alternatively accept a GenerateContentRequest.
```

Comment:
Nit: backticks on `countTokens` and `GenerateContentRequest`

Reply:
Done.